cedar-policy-cli/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
   (previously required). If not provided, the linked policy will only be shown on
   stdout; if it is provided, the indicated file will be updated with the new link
   (as before).
+- Added support for the human-readable schema format (`--schema-format human` when a schema is needed).
```

Comment:
Do we want to call more attention to this? Or keep the changelog short?

Reply:
I think I'll move it to the top per Kesha's suggestion. And probably elaborate it a little bit.